Snakefile

```diff
@@ -77,6 +77,16 @@ rule build_simplified_population_layouts:
     benchmark: "benchmarks/build_clustered_population_layouts/s{simpl}"
     script: "scripts/build_clustered_population_layouts.py"
 
+rule build_gas_network:
+    input:
+        gas_network="data/gas_network/gas_network_dataset.csv",
```

Comment:
How is this dataset created from SciGrid gas and ENTSOG? Should we include the scripts to do that such that we have the full pipeline from raw data to the model?

Reply:
We can, it probably would need to be a bit cleaned, it is currently in this [repository](https://github.com/ericzhou571/salt_cave-gas_network/blob/main/script/build_gas_network.py). We were thinking about keeping it separate, so that one could use the data also in another model. But maybe it is easier to move it also to pypsa-eur-sec.

Reply:
Ah, this also has the hydrogen storage stuff. Nice! I think we should move all of it to PyPSA-Eur. Otherwise, we create more release dependencies. For other's to use these datasets, we could think about publishing the rule outputs in separate zenodo repositories once it's all done.

Reply:
we'll do this in #153